tools/src/main/java/org/apache/kafka/tools/CompatibilityTest.java

```diff
@@ -93,22 +93,22 @@ public static void main(String[] args) throws Exception {
                 System.exit(1);
             }
         }
-        String bootstrapServers = res.getString("bootstrap_servers");
+        String bootstrapServer = res.getString("bootstrap_server");
```

Comment:
It's just the name of the metavar.  The actual command line option is still --bootstrap-server.  Confusing, I know, but dashes in argparse4j metavars don't seem to work (curious if anyone has any more information on this limitation)